graphics/digital_differential_analyzer_line_drawing.py

```diff
@@ -0,0 +1,69 @@
+import matplotlib.pyplot as plt
+from typing import list, tuple
+
+
+def digital_differential_analyzer_line(
+    x1: int, y1: int, x2: int, y2: int
+) -> tuple[list[int], list[int]]:
+    """
+    Draw a line using the Digital Differential Analyzer (DDA) algorithm.
+
+    Parameters:
+        x1 (int): The x-coordinate of the starting point.
+        y1 (int): The y-coordinate of the starting point.
+        x2 (int): The x-coordinate of the ending point.
+        y2 (int): The y-coordinate of the ending point.
+
+    Returns:
+        tuple of two lists, where the first list contains the x-coordinates of the line points,
+        and the second list contains the y-coordinates.
```

Comment:
Instead of having separate lists for the x- and y-coordinates, could you have the algorithm just return a list of coordinate pairs?